pkg/config/configuration.go

```diff
@@ -84,6 +85,7 @@ type ConfigurationSpec struct {
 	MetricSpec        MetricSpec        `json:"metric,omitempty" yaml:"metric,omitempty"`
 	Secrets           SecretsSpec       `json:"secrets,omitempty" yaml:"secrets,omitempty"`
 	AccessControlSpec AccessControlSpec `json:"accessControl,omitempty" yaml:"accessControl,omitempty"`
+	FeatureSpec       []FeatureSpec     `json:"features,omitempty" yaml:"features,omitempty"`
```

Comment:
Just `Features`, similar to `Secrets`. We should probably remove the `Spec` suffix from the other attributes too.

Reply:
Done and I agree they should be normalized. I can through that in here but would prefer a separate PR for it.